_component_design/popovers.md

```diff
@@ -14,81 +14,183 @@ last-modified: 2017-08-17
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Meet the popover pattern"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/content-areas/popovers/popover-hero-image.svg){:width="100%"}
 {% endfigure %}
 {% endcolumn %}
-</div>
 
-## Triggered Popover
+</div>
 
+## Default Layout 
+{: .hxSectionTitle}
 <div class="hxRow">
-{% column left:"hxCol-4" %}
-Overpopovers are triggered as a result of a user hovering over linked area in
-the UI.
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+The most defining aspects of a popover are the container and the pointer. The container  appears on top of the canvas. The pointer points to the popover trigger. Pointers may be used on either side of the four corners.
 
-- font-family: Roboto
-- font-weight: 400
-- font-size: 16px / 1em
-- color: Grey800 (#616161)
+- Popovers **should only be shown** when a trigger is clicked.
+- Most popovers may also have a heading and submit/cancel buttons. If buttons are not needed, a popover should close when the user clicks outside of the popover.
+- The content area may include text, inputs, or interactive content like a table or visualization.
 
 {% endcolumn %}
-{% column left:"hxCol-8" %}
-{% figure [caption:"Triggered popover"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/Editing/popover-triggered.svg){:width="100%"}
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"A popover consists of a panel, contextual pointer, heading and buttons."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/content-areas/popovers/popover-default-layout.svg){:width="80%"}
 {% endfigure %}
-{% figure [caption:"Triggered popover specs"] [class:"image bg-light border"] %}
-![]({{site.cdn_url}}/img/components/Editing/popover-triggered-specs.svg){:width="100%"}
+{% endcolumn %}
+</div>
+
+## Pattern Specifications 
+{: .hxSectionTitle}
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Popover Spacing
+{: .hxSubSectionTitle}
+For specs on laying out form fields and body text within a popover, refer to the following patterns:
+
+- <a href="{{site.baseurl}}/components/checkboxes.html">Checkboxes</a>
+- <a href="{{site.baseurl}}/components/dropdowns.html">Dropdowns</a>
+- <a href="{{site.baseurl}}/components/radio-buttons.html">Radio Buttons</a>
+- <a href="{{site.baseurl}}/components/type-inputs.html" target="_blank">Text Inputs</a>
+- <a href="{{site.baseurl}}/style/typography.html">Typography</a>
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Popover spacing specifications"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/content-areas/popovers/popover-spacing.svg){:width="80%"}
```

Comment:
This SVG has some issues with redline labels.  I re-exported it and emailed it to you.

Reply:
Unsure what the issue with the SVG's was, but I am pushing the 5 SVGs you sent my way via Email in the newest commit to the PR.